packages/elements/src/components/ino-carousel/ino-carousel.tsx

```diff
@@ -144,6 +153,33 @@ export class InoCarousel implements ComponentInterface {
     }
   };
 
+  private emitSlideChange(iconArrow: ArrowDirections): void {
+    const carouselSlide = this.getSlides()[this.currentSlide];
+    const allSlides = this.getSlides();
+
+    if (!carouselSlide && allSlides[0] !== undefined) {
+      this.valueChange.emit(allSlides[0].value);
+      return;
+    }
+
+    let nextSlide: HTMLInoCarouselSlideElement;
+    switch (iconArrow) {
+      case ArrowDirections.RIGHT:
+        nextSlide =
+          (carouselSlide.nextElementSibling as HTMLInoCarouselSlideElement) ??
+          allSlides[0];
+        break;
+      case ArrowDirections.LEFT:
+        if (allSlides.length !== 0) {
+          nextSlide =
+            (carouselSlide.previousElementSibling as HTMLInoCarouselSlideElement) ??
+            allSlides[allSlides.length - 1];
+        }
```

Comment:
We've confirmed that all slides is not empty
```suggestion
	      nextSlide =
	        (carouselSlide.previousElementSibling as HTMLInoCarouselSlideElement) ??
	        allSlides[allSlides.length - 1];
```